calcom/cal.com
# fix Former time in wrong time zone (Rescheduling)

## What does this PR do?
This PR fixes the former time bug in the wrong time zone while rescheduling a meeting.


fixes: #9024 

## Type of change



- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
- book a meeting
- reschedule the meeting while doing so make sure to change the timezone
- see the former time changes according to your new booking timezone.

## Checklist




- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
